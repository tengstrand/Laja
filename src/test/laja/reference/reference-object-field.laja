
#set (integer = 1)
#set (myAttributes = new net.sf.laja.integrationtest.ReferenceTest$MyAttributes())

#write "target/test-output/reference-object-field.out"
  integer.MAX_VALUE = {integer.MAX_VALUE}
  myAttributes.x = {myAttributes.x}
  myAttributes.y = {myAttributes.y}
  myAttributes.z.length = {myAttributes.z.length}
#end
