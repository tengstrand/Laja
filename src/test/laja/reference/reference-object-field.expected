  integer.MAX_VALUE = 2147483647
  myAttributes.x = 123
  myAttributes.y = abc
  myAttributes.z.length = 1
